fastapi/app/main.py

```
@@ -119,117 +56,154 @@ async def process_video(file: UploadFile = File(...)):
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE
             )
-            if result.returncode != 0:
-                raise RuntimeError(f"FFmpeg 오류: {result.stderr}")
+            image_data = thumb_proc.stdout
+            if image_data:
+                files = {
+                    "file": ("thumbnail.jpg", image_data, "image/jpeg")
+                }
+                headers = {
+                    "Authorization": f"Bearer {{your_token_here}}"  # 필요 시 토큰 삽입
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)